### PR TITLE
Implement twig hooks for account address_book add and edit pages

### PR DESCRIPTION
**Pull request by bartek-sek**
image

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/shared/form/address.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+twig_hooks:
+    hooks:
+        'sylius_shop.shared.form.address':
+            name_row:
+                template: '@SyliusShop/shared/form/address/name_row.html.twig'
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/shared/form/address.yaml]:
```suggestion
            name:
                template: '@SyliusShop/shared/form/address/name.html.twig'
```
👀 ?

File: src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig [hunk 2]
```diff
@@ -13,17 +10,6 @@
 
 {% block account_content %}
     <div>
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig]: Is this empty `<div>` needed here?

File: src/Sylius/Bundle/ShopBundle/templates/account/address_book/update.html.twig [hunk 3]
```diff
@@ -13,17 +10,6 @@
 
 {% block account_content %}
     <div>
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/templates/account/address_book/update.html.twig]: As above

File: src/Sylius/Bundle/ShopBundle/templates/shared/account/address_book/form/fields.html.twig [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<div class="mb-4">
+    {% hook 'address' with { _prefixes: ['sylius_shop.shared.form'] } %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/templates/shared/account/address_book/form/fields.html.twig]: Are the custom (for checkout and address book) hooks also triggered here or somewhere? 🤔

File: src/Sylius/Bundle/ShopBundle/templates/shared/form/address/city_row.html.twig [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<div>
+    {% hook 'city_row' %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/templates/shared/form/address/city_row.html.twig]: Why not?:
```suggestion
    {% hook 'city' %}
```

File: src/Sylius/Bundle/ShopBundle/templates/shared/form/address/name_row.html.twig [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<div class="row">
+    {% hook 'name_row' %}
```
Comment (GSadee) [inline src/Sylius/Bundle/ShopBundle/templates/shared/form/address/name_row.html.twig]:
```suggestion
    {% hook 'name' %}
```
🤔

File: src/Sylius/Bundle/ShopBundle/templates/shared/account/address_book/form/fields.html.twig (hunk 4 above)
Comment (bartek-sek) [reply src/Sylius/Bundle/ShopBundle/templates/shared/account/address_book/form/fields.html.twig]: No they were not, but I fixed it and now it works properly

**Review by TheMilek — APPROVED** [review]

**Review by mpysiak — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Pull request merged**